src/canopy/chat_engine/query_generator/last_message.py

```diff
@@ -0,0 +1,17 @@
+from typing import List
+
+from canopy.chat_engine.query_generator import QueryGenerator
+from canopy.models.data_models import Messages, Query
+
+
+class LastMessageQueryGenerator(QueryGenerator):
+
+    def generate(self,
+                 messages: Messages,
+                 max_prompt_tokens: int) -> List[Query]:
+        return [Query(text=messages[-1].content)]
```

Comment:
Consider raise a value error for empty messages list with an explanation to the user

Reply:
Done